Cargo.toml

```diff
@@ -14,18 +14,24 @@ rust-version = "1.67.0"
 
 [[example]]
 name = "simple"
-required-features = ["inspect"]
 
 [[example]]
 name = "multiple"
-required-features = ["inspect"]
 
 [features]
-default = ["serialize"]
-inspect = ["bevy-inspector-egui"]
```

Comment:
Try not to modify the rest of the code, keep this feature so that there are no conflicts with the other PR's.